src/server.rs

```diff
@@ -973,33 +975,43 @@ impl LanguageServer for LspServer {
         };
         Ok(Some(response))
     }
-    async fn references(
+    async fn document_highlight(
```

Comment:
Nit: We should add an empty line between each function  so that the git diff doesn't get messed up like this.